proto/osmosis/tokenfactory/v1beta1/tx.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+package osmosis.tokenfactory.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v7/x/tokenfactory/types";
+
+// Msg defines the Msg service.
+service Msg {
+  rpc CreateDenom(MsgCreateDenom) returns (MsgCreateDenomResponse);
+  rpc Mint(MsgMint) returns (MsgMintResponse);
+  rpc Burn(MsgBurn) returns (MsgBurnResponse);
+  // ForceTransfer is deactivated for now because we need to think through edge
+  // cases rpc ForceTransfer(MsgForceTransfer) returns
+  // (MsgForceTransferResponse);
+  rpc ChangeAdmin(MsgChangeAdmin) returns (MsgChangeAdminResponse);
+}
+
+// ===================== MsgCreateDenom
```

Comment:
nit: IMO I find these comment blocks unhelpful and kinda annoying. Can we just stick with simple docs above the type?

```suggestion
```